test/systemtest/test_separate_pkgs.py

```diff
@@ -135,6 +135,16 @@ def test_pkg_spdx_tag(self):
         self.assertEqual(os.EX_OK, main(
             ["test/_test_data/test_pkg_spdx_tag"]))
 
+    def test_pkg_too_many_license_files(self):
+        """"Test on a package with multiple License files that are not
+        declared by any tag and could therefore be removed."""
+        process, stdout = open_subprocess("test_pkg_too_many_license_files")
+        self.assertEqual(os.EX_OK, process.returncode)
```

Comment:
Why did we say this is a warning?
I would argue, its an error. This could indicate
- The package is issues under multiple licenses
- There are licenses that are not declared in the package.xml
An these are things we want to catch.